tests/test_batoid_wcs.py

```diff
@@ -463,33 +462,37 @@ def test_config():
     camera = imsim.get_camera()
 
     rotTelPos =  cmds['rottelpos'] * galsim.degrees
-
+    telescope = imsim.load_telescope(f"LSST_{band}.yaml", rotTelPos=rotTelPos)
     # Non-default values.
     temperature = 293.
     pressure = 69.0
     H2O_pressure = 2.0
 
     factory = imsim.BatoidWCSFactory(
-        boresight, rotTelPos, obstime, fiducial_telescope, wavelength,
+        boresight, obstime, telescope, wavelength,
         camera,
         temperature=temperature,
         pressure=pressure,
         H2O_pressure=H2O_pressure
     )
 
     config = {
+        'input': {
+            'telescope': {
+                'file_name':f"LSST_{band}.yaml",
+                'rotTelPos': {'type': 'Degrees', 'theta':rotTelPos.deg}
```

Comment:
This is fine, but you could also just write
```
'rotTelPos': rotTelPos
```
In tests, you don't necessarily need to drop all the way back to a string-able yaml.  You can just give a complete object if you already have it.